tests/integration/fileTests.py

```diff
@@ -341,6 +343,14 @@ def test_check_status(self):
             self.assertEqual(json["created_on"], datetime.utcnow().strftime("%m/%d/%Y"))
             self.assertEqual(json["last_updated"], submission.updated_at.strftime("%Y-%m-%dT%H:%M:%S"))
 
+    def test_get_obligations(self):
```

Comment:
Thanks for being a great test case re: the unit vs integration test conversation!